projects/js-packages/charts/src/components/line-chart/line-chart.tsx

```diff
@@ -273,16 +321,81 @@ const LineChart: FC< LineChartProps > = ( {
 		yAccessor: ( d: DataPointDate ) => d?.value,
 	};
 
+	// Create a custom renderTooltip that includes focus capability
+	const focusableRenderTooltip = useMemo( () => {
+		return ( params: RenderTooltipParams< DataPointDate > ) => {
+			const tooltipContent = renderTooltip( params );
+
+			if ( selectedIndex !== undefined ) {
+				return (
+					<div
+						ref={ tooltipRef }
+						tabIndex={ -1 }
+						role="tooltip"
+						aria-atomic="true"
+						className={ styles[ 'line-chart__tooltip--keyboard-focused' ] }
+						data-testid={ `line-chart-tooltip-${ selectedIndex }` }
+						key={ `line-chart-tooltip-${ selectedIndex }` }
+					>
+						{ tooltipContent }
+					</div>
+				);
+			}
+
+			return (
+				<div role="tooltip" aria-live="polite">
+					{ tooltipContent }
+				</div>
+			);
+		};
+	}, [ renderTooltip, selectedIndex, tooltipRef ] );
+
+	const onKeyDown = useMemo(
+		() => ( event: React.KeyboardEvent< HTMLDivElement > ) => {
+			const size = dataSorted[ 0 ]?.data.length || 0;
+			if ( size === 0 ) return;
+
+			const currentSelectedIndex = selectedIndex === undefined ? -1 : selectedIndex;
+
+			if ( currentSelectedIndex + 1 >= size && [ 'ArrowRight', 'Tab' ].includes( event.key ) ) {
+				chartRef.current?.focus();
+
+				setSelectedIndex( undefined );
+				return;
+			}
+
+			event.preventDefault();
+
+			if ( [ 'ArrowRight', 'Tab' ].includes( event.key ) ) {
```

Comment:
[I was reading this](https://github.com/recharts/recharts/wiki/Recharts-and-accessibility#recharts-uses-arrow-keys-to-navigate-between-data-points) and agree that tabbing through all the points even though you don't care will be annoying. I will change to support only arrows to navigate data points. let me know if anyone has any thoughts

Reply:
Yes, 💯 agree. Only the Chart should be in the tab index so a keyboard user can easily step over it.

Reply:
Agree! And thanks for linking the reference.

Reply:
I have handled this in the latest commits in this PR